recipes/pymochi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.9" %}
+{% set github = "https://github.com/lehner-lab/MoCHI" %}
+
+package:
+  name: pymochi
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/lehner-lab/MoCHI/archive/v{{ version }}.tar.gz
+  sha256: cec741cc8cda6affc9f4c2fd470afd28ee7c5ec9282bd59a4d4102a6e3afe005
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install ./ --ignore-installed --no-deps -vv
+  run_exports:
+    - {{ pin_subpackage('pymochi', max_pin="x.x") }}
```

Comment:
| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

Reply:
@thanhleviet I'm not sure how to choose the appropriate statement here - could you advise please?

Reply:
If there are no breaks in the minor version, probably choose `max_pin="x"`